cmd/kubeadm/app/constants/constants.go

```diff
@@ -165,6 +165,9 @@ var (
 	// UseEnableBootstrapTokenAuthFlagVersion defines the first version where the API server supports the --enable-bootstrap-token-auth flag instead of the old and deprecated flag.
 	// TODO: Remove this when the v1.9 cycle starts and we bump the minimum supported version to v1.8.0
 	UseEnableBootstrapTokenAuthFlagVersion = version.MustParseSemantic("v1.8.0-beta.0")
+
+	// MinimumKubeletVersion specifies the minimum version of kubelet which kubeadm supports
+	MinimumKubeletVersion = version.MustParseSemantic("v1.8.0-alpha.3")
```

Comment:
The minimum kubelet version v1.9 CLI is gonna support is `v1.8.0`.
You can add a comment that this is currently `(CLI_VERSION-1)`, but we might relax this to `(CLI_VERSION-2)` in the future

Reply:
That's minimum reasonable default at the moment. We can put even 1.8.0-rc.1 now, to be able to test and deploy at least something reasonable until real v1.8.0 released.
In longer term, I'd like to re-factor all the pieces where it checks different versions, so it will be generic X-1 instead of constants. But again, better in separate PR this kind of cleanups.